dependencies/libics/CMakeLists.txt

```diff
@@ -144,79 +144,80 @@ endif()
 
 
 # Unit tests
-enable_testing()
-
-add_executable(test_ics1 EXCLUDE_FROM_ALL test_ics1.c)
-target_link_libraries(test_ics1 libics)
-add_executable(test_ics2a EXCLUDE_FROM_ALL test_ics2a.c)
-target_link_libraries(test_ics2a libics)
-add_executable(test_ics2b EXCLUDE_FROM_ALL test_ics2b.c)
-target_link_libraries(test_ics2b libics)
-if(LIBICS_USE_ZLIB)
-   add_executable(test_gzip EXCLUDE_FROM_ALL test_gzip.c)
-   target_link_libraries(test_gzip libics)
+option(ENABLE_ICS_TESTS "Enable libics tests" OFF)
```

Comment:
This is a good idea. I guess it's interfering with your own tests?

Actually, I'd be happy also to just remove the tests from this directory, leave only the necessary library code. DIPlib itself does testing of ICS reading and writing, so these tests are mostly redundant anyway.

I would encourage you to submit these changes to [the upstream repository](https://github.com/svi-opensource/libics).